cms/templates/cms/home_page.html

```diff
@@ -217,31 +165,80 @@ <h3>Why should I enroll?</h3>
                <img src="{% static 'images/online.png' %}" alt="">
             </div>
 
-            <h4>Convenient &amp; Flexible</h4>
-            <p>Take online courses on a schedule that fits your needs, and meet motivated learners who share your interests.</p>
+            <h4>Advance your career</h4>
```

Comment:
```suggestion
            <h4>Advance Your Career</h4>
```
Title Case for all of these H4s